emcc.py

```diff
@@ -2295,6 +2325,18 @@ def post_link(options, in_wasm, wasm_target, target):
         minified_worker = building.acorn_optimizer(worker_output, ['minifyWhitespace'], return_output=True)
         open(worker_output, 'w').write(minified_worker)
 
+    if shared.Settings.USE_WASM_WORKERS == 1:
+      worker_output = os.path.join(target_dir, shared.Settings.WASM_WORKER_FILE)
+      with open(worker_output, 'w') as f:
+        f.write(shared.read_and_preprocess(shared.path_from_root('src', 'wasm_worker.js'), expand_macros=True))
```

Comment:
* We need to generate a startup bootstrap script for the Workers (people could be asked to put this together themselves separate to the build process I suppose)